src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -821,11 +870,11 @@ public void TestArrayContainingLambdaAsVariable()
         public void TestLambdaIsCalledOnce()
         {
             int global = 0;
-            _context["callcount"] = (Proc) (c =>
-            {
-                ++global;
-                return global.ToString();
-            });
+            _context["callcount"] = (Proc)(c =>
+           {
+               ++global;
+               return global.ToString();
+           });

```

Comment:
Seems to be incorrect change - 873 has 12 spaces and 874 has 11.

Reply:
Fixed.